cmd/compareconfig.go

```diff
@@ -0,0 +1,206 @@
+package main
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"sigs.k8s.io/yaml"
+
+	"github.com/Azure/ARO-RP/pkg/deploy"
+	"github.com/Azure/ARO-RP/pkg/util/cmp"
+)
+
+func main() {
+	opts := DefaultOptions()
+	cmd := &cobra.Command{
+		Use:          "overview",
+		Short:        "overview",
+		Long:         "overview",
+		SilenceUsage: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return compare(cmd.Context(), opts)
+		},
+	}
+	if err := BindOptions(opts, cmd); err != nil {
+		slog.Error("failed to bind options", "error", err)
+		os.Exit(1)
+	}
+
+	if err := cmd.Execute(); err != nil {
+		slog.Error("failed to compare configs", "error", err)
+		os.Exit(1)
+	}
+}
+
+func compare(ctx context.Context, opts *RawOptions) error {
+	validated, err := opts.Validate()
+	if err != nil {
+		return err
+	}
+	completed, err := validated.Complete()
+	if err != nil {
+		return err
+	}
+	if diff := cmp.Diff(completed.ClassicConfig, completed.RAConfig); diff != "" {
+		fmt.Println(diff)
```

Comment:
@stevekuznetsov you want to print this diff?

Reply:
Yeah, whoever is running this will need it to figure out what's wrong :)